docs/shipping/Compute/winlogbeat.md

```diff
@@ -20,7 +20,147 @@ drop_filter: []
 [Winlogbeat 8](https://www.elastic.co/guide/en/beats/winlogbeat/8.7/winlogbeat-installation-configuration.html#installation), [Winlogbeat 7](https://www.elastic.co/guide/en/beats/winlogbeat/7.x/winlogbeat-installation-configuration.html#installation), or [Winlogbeat 6](https://www.elastic.co/guide/en/beats/winlogbeat/6.8/winlogbeat-installation.html).
 
  
+## Manually configure OpenTelemetry on Localhost
```

Comment:
Not really sure how the agent is relevant in Winlogbeat... they also don't need Winlogbeat to send logs if they use the agent...

Reply:
So should I just remove it? I thought this is the "Windows localhost" page? 🤔  @ShiranAvidov